_utility-aliases.scss

```diff
@@ -55,77 +55,85 @@ $text-color--positive: $color--fa-green-text;
 
 /* Color map */
 $color-values: (
-  alt:           $color--alt,
-  base:          $color--base,
-  fa-amber:      $color--fa-amber,
-  fa-blue:       $color--fa-blue,
-  fa-blue-dark:  $color--fa-blue-dark,
-  fa-blue-light: $color--fa-blue-light,
-  fa-cyan:       $color--fa-cyan,
-  fa-green:      $color--fa-green,
-  fa-green-dark: $color--fa-green-dark,
-  fa-orange:     $color--fa-orange,
-  fa-pink:       $color--fa-pink,
-  fa-purple:     $color--fa-purple,
-  fa-red:        $color--fa-red,
-  fa-red-orange: $color--fa-red-orange,
-  fa-yellow:     $color--fa-yellow,
-  gray-1:        $color--gray-1,
-  gray-2:        $color--gray-2,
-  gray-3:        $color--gray-3,
-  gray-4:        $color--gray-4,
-  gray-5:        $color--gray-5,
-  gray-6:        $color--gray-6,
-  gray-7:        $color--gray-7,
-  gray-8:        $color--gray-8,
-  gray-9:        $color--gray-9,
-  gray-10:       $color--gray-10,
-  gray-11:       $color--gray-11,
-  gray-12:       $color--gray-12,
-  gray-13:       $color--gray-13,
-  gray-14:       $color--gray-14,
-  gray-15:       $color--gray-15,
-  gray-16:       $color--gray-16
+  alt:                    $color--alt,
+  base:                   $color--base,
+  fa-amber:               $color--fa-amber,
+  fa-blue:                $color--fa-blue,
+  fa-blue--shade-1:       $color--fa-blue--shade-1,
+  fa-blue--shade-2:       $color--fa-blue--shade-2,
+  fa-blue--shade-3:       $color--fa-blue--shade-3,
```

Comment:
I agree with you on the problem of numbered values proliferating, but I'm not sure that the `xx-small` > `xx-large` pattern is necessarily ideal here either.

I don't think we should imply the existence of a pre-calculated scale because all the colours have to be manually mixed. Even 7 variants of each base colour is approaching 100 colours, most of which will never get used anywhere. I'd rather have as few colours as possible and encourage  adding new ones as they are needed.